flo_ai/router/flo_llm_router.py

```diff
@@ -118,4 +120,5 @@ def build(self):
                 flo_team=self.flo_team,
                 name=self.name,
                 session=self.session,
+                model_name=self.model_name, 
```

Comment:
Same as above FloLLMRouter has to accept the model name

Reply:
resolved